docs/parallel_training/recommended_pratice.rst

```diff
@@ -0,0 +1,75 @@
+Recommended Practice
```

Comment:
spelling error in filename: pratice -> practice

Reply:
done